src/java/org/apache/cassandra/repair/autorepair/AutoRepair.java

```diff
@@ -449,20 +449,27 @@ private List<String> retrieveTablesToBeRepaired(Keyspace keyspace, AutoRepairCon
     }
 
     private void cleanupAndUpdateStats(RepairTurn turn, AutoRepairConfig.RepairType repairType, AutoRepairState repairState, UUID myId,
-                                       long startTime, CollectedRepairStats collectedRepairStats) throws InterruptedException
+                                       long startTimeInMillis, CollectedRepairStats collectedRepairStats) throws InterruptedException
     {
         //if it was due to priority then remove it now
         if (turn == MY_TURN_DUE_TO_PRIORITY)
         {
             logger.info("Remove current host from priority list");
             AutoRepairUtils.removePriorityStatus(repairType, myId);
         }
-
+        long repairScheduleElapsedInMillis = timeFunc.get() - startTimeInMillis;
+        if (repairScheduleElapsedInMillis < SLEEP_IF_REPAIR_FINISHES_QUICKLY.toMilliseconds())
+        {
+            //If repair finished quickly, happens for an empty instance, in such case
```

Comment:
What makes an instance empty in this context?

Reply:
It's the terminology we use internally. _instance=Cassandra cluster_ :) 
Updated the comment to reflect an empty Cassandra cluster with empty tables (or tiny data)

Reply:
thanks for clearing that up and updating the comment :+1: